*: increase the performance of loading stats (#17895)
pingcap/docs: tidb-configuration-file.md

### `stats-load-concurrency` <span class="version-mark">New in v5.4.0</span>

+ The maximum number of columns that the TiDB synchronously loading statistics feature can process concurrently.

> + Default value: `0`, prior to v8.2, the default value was 5.
> + Currently, the valid value range is `[0, 128]`, with 0 being the automatic mode, which adjusts concurrency based on the machine's configuration. Before v8.2, the minimum value was 1.

Reviewer: ```suggestion
+ Default value: `0`. Before v8.2.0, the default value is 5.
+ Currently, the valid value range is `[0, 128]`. The value 0 means the automatic mode, which adjusts concurrency based on the configuration of the server. Before v8.2.0, the minimum value is 1.
```
